.github/workflows/ci.yml

```diff
@@ -5,7 +5,6 @@ on:
       - 'master'
       - 'release-[0-9]+.[0-9]+'
   pull_request:
-    types: [opened, synchronize, reopened, ready_for_review]
```

Comment:
Why remove this?

Reply:
So the CI isn't rerun when a contributor click ready for review.

Reply:
Does `types` have a default value?

Reply:
Yeah, `opened`, `synchronize` and `reopened`